### PR TITLE
Add man page and fix typo for update command

### PR DESCRIPTION
**Pull request by hqhq**
Signed-off-by: Qiang Huang <h.huangqiang@huawei.com>

### CODE REVIEW

**Comment by cyphar** [comment]
LGTM.

File: man/runc-update.8.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+       "blkioWeight": 0
+     },
+   }
+
+Note: if data is to be read from a file or the standard input, all
+other options are ignored.
+
+# OPTIONS
+   --resources, -r         path to the file containing the resources to update or '-' to read from the standard input.
+   --blkio-weight "0"      Specifies per cgroup weight, range is from 10 to 1000.
```
Comment (mrunalp) [inline man/runc-update.8.md]: nit: Take out the double quotes around 0 as it implies a string.

Comment (hqhq) [inline man/runc-update.8.md]: It's copied from `runc update --help` and I think it's a cli problem, shall we address this in a separate PR?

Comment (mrunalp) [inline man/runc-update.8.md]: Sure

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**